src/keychain/index.js

```diff
@@ -1,3 +1,469 @@
+/* eslint max-nested-callbacks: ["error", 5] */
 'use strict'
 
-module.exports = require('./keychain')
```

Comment:
content of `src/keychain/keychain` was moved here to be consistent with the rest of the libp2p codebase